src/ProjectConfiguration.ts

```diff
@@ -135,6 +136,11 @@ const permissionTypes = [
     description:
       'Companion may communicate with the Internet using your phone data connection',
   },
+  {
```

Comment:
Place this in the private list below to stop it appearing in Studio for non @fitbit.com users.

Reply:
Thanks, will do!

Reply:
Thanks, will do!